apps/prairielearn/src/migrations/20230715221742_PLR_seasonal_session.sql

```diff
@@ -1,15 +1,15 @@
 -- SEASONAL_SESSION
 CREATE TABLE IF NOT EXISTS PLR_seasonal_session (
     id SERIAL PRIMARY KEY,
-    course_instance_id BIGINT NOT NULL
+    course_instance_id BIGINT NOT NULL UNIQUE
     -- FOREIGN KEY (course_instance_id) REFERENCES course_instances (id)
 );
 
 -- SEASONAL_SESSION_CREDENTIALS
 CREATE TABLE IF NOT EXISTS PLR_seasonal_session_CREDENTIALS (
     id SERIAL PRIMARY KEY,
-    session_id INT NOT NULL,
     user_id BIGINT NOT NULL,
+    session_id INT NOT NULL,
```

Comment:
Again, `NOT NULL` might cause some issues if we're using the PL population scripts, but we could also make our own to simulate it.

Reply:
Not null was not added here, I just shifted session_id down so that it was the same order as live session.